code/modules/multiz/structures.dm

```diff
@@ -26,9 +26,12 @@
 	if(anchored != last_anchored)
 		find_connections()
 
-/obj/structure/ladder/Initialize(var/ml, var/_mat)
-	. = ..()
-	if(ml)
+/obj/structure/ladder/Initialize(maploading, material)
+	..()
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/structure/ladder/LateInitialize(maploading, material)
```

Comment:
While it doesn't matter here, I feel it's correct to always call parent in this proc.

Reply:
Parent proc doesn't do anything though.

Reply:
Yes, but if something is added to it (and with the New -> Initialize enforcement that's going to become more and more likely) this would then not see that. Of course here it doesn't matter; I'm suggesting that "Always call parent in LateInitialize" should be something to enforce generally, like "Always return `..()` in examine".

Reply:
I don't mind either way but unlike `/New()` and `/Initialize()` the base of `/LateInitialize()` doesn't do anything of importance